src/stirling/utils/detect_application.cc

```diff
@@ -49,6 +49,17 @@ Application DetectApplication(const std::filesystem::path& exe) {
   return Application::kUnknown;
 }
 
+std::string GetNodeApplicationFilename(std::string_view cmdline) {
```

Comment:
Maybe include a sample cmdline here for reference.

Reply:
Doc string with example added in https://github.com/pixie-io/pixie/pull/1971/commits/92f79bac649483a2caec9dddc4b0dcb003e16e1c.